mex/common/backend_api/connector.py

```diff
@@ -56,7 +56,7 @@ def post_models(
         Returns:
             Identifiers of posted extracted models
         """
-        # XXX deprecated method, please use `post_extracted_models` instead
+        # XXX deprecated method, please use `post_extracted_items` instead
```

Comment:
that's sadly only in python 3.13

Reply:
Meh. How about at least raising a warning? I fear that this deprecation will be overlooked otherwise. Or do we get some kind of warning if code uses this? I leave it up to you.

Reply:
i removed it fully instead 794f7ab3ef0f5e8586c37b2bce2d34dca9255c0f